prody/atomic/bond.py

```diff
@@ -15,23 +15,32 @@ class Bond(object):
     * :func:`len` returns bond length, i.e. :meth:`getLength`
     * :func:`iter` yields :class:`~.Atom` instances"""
 
-    __slots__ = ['_ag', '_acsi', '_indices']
+    __slots__ = ['_ag', '_acsi', '_indices', '_bondOrder']
+    _bondType = {1:'single', 2:'double', 3:'triple', 4:'aromatic', 5:'amid', }
```

Comment:
I assume there's a typo here and amide needs an e

Reply:
yes, corrected typo